api/src/context-api.ts

```diff
@@ -14,6 +14,16 @@
  * limitations under the License.
  */
 
-/** only globals that common to node and browsers are allowed */
-// eslint-disable-next-line node/no-unsupported-features/es-builtins
-export const _globalThis = typeof globalThis === 'object' ? globalThis : global;
+export {
+  createContextKey,
+  ROOT_CONTEXT,
+} from './context/context';
+export {
+  Context,
+  ContextManager,
+} from './context/types';
+
+import { ContextAPI } from './api/context';
+export type { ContextAPI } from './api/context';
+/** Entrypoint for context API */
+export const context = ContextAPI.getInstance();
```

Comment:
Can you add a comment to this file that explains it must be kept separate from the index in order to allow for proper tree shaking? I worry we may forget why we did this in the future (or if the project gets new maintainers and we're gone)

Reply:
I did comment in the api/src/index.ts: https://github.com/open-telemetry/opentelemetry-js/pull/3374/files#diff-c7daeed68470fce500b64961dc1986f90d3c903d7679f553701de7aed7aa72a6R33.

Do you prefer adding the comment in every single API index file?

Reply:
I missed that comment. if you think that is enough then fine. Just want to make sure it is somewhere